### PR TITLE
Add Cloudflare Pages support to README

### PR DESCRIPTION
**Pull request by rita3ko**
Demo: https://my-zola-project.pages.dev/ !

**IMPORTANT: Please do not create a Pull Request adding a new feature without discussing it first.**

The place to discuss new features is the forum: <https://zola.discourse.group/>
If you want to add a new feature, please open a thread there first in the feature requests section.

Sanity check:

* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/getzola/zola/pulls) for the same update/change?

## Code changes
(Delete or ignore this section for documentation changes)

* [ ] Are you doing the PR on the `next` branch?

If the change is a new feature or adding to/changing an existing one:

* [ ] Have you created/updated the relevant documentation page(s)?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ in the `docs/content` folder of the repository and the community can use [its fo
 | LiveReload                      | ![yes] | ![no]  | ![yes] | ![yes]  |
 | Netlify support                 | ![yes] | ![no]  | ![yes] | ![no]   |
 | Vercel support                  | ![yes] | ![no]  | ![yes] | ![yes]  |
+| Cloudflare Pags support         | ![yes] | ![no]  | ![yes] | ![yes]  |
```
Comment (Keats) [inline README.md]: pages

**Review by Keats — CHANGES_REQUESTED** [review]
How does it work? Could you add some docs for it?

**Comment by codeandmedia** [comment]
Sorry to interrupt you, I just have tested Pages, it looks pretty easy and tremendously fast. It's a worker on Cloudflare infrastructure that contains Zola binary and upload an output to CF storage. Cloudflare have HowTo here: https://developers.cloudflare.com/pages/how-to/deploy-a-zola-site . 

@Rita3ko first of all, many thanks Cloudflare team for this cool service. I guess it would be useful to add Cloudflare here (https://www.getzola.org/documentation/deployment/overview/) and we already have all what it needed, some screenshots might improved the text. I can help with that, but don't know how to do it properly.

**Comment by Keats** [comment]
@codeandmedia could you do a PR explaining how to get set up in the docs? Even if it's just a link to the cloudflare page

**Comment by codeandmedia** [comment]
> 
> 
> @codeandmedia could you do a PR explaining how to get set up in the docs? Even if it's just a link to the cloudflare page

done! https://github.com/getzola/zola/pull/1414

**Pull request merged**